Add management command for populating calculation_logic on Figure model (#572)
idmc-labs/helix-server: README.md

```bash
python manage.py addstatictoken -t 123456 "admin@idmcdb.org"
```

> # Management Command
> A custom management command file has been added to the project to facilitate specific tasks. To use this command:
> 
> ```bash
> python manage.py populate_calculation_logic_field
> ```
> 
> This command populates the `calculation_logic` field in the Figure Table on the Figure table if there is no existing data in it.

Reviewer: ````suggestion
## Management Command
There are custom management commands available to facilitate specific tasks.

### Populate figure `Calculation Logic`
```bash
./manage.py populate_calculation_logic_field
```
> NOTE: This command populates the `calculation_logic` field in the Figure Table if there is no existing data in it.

````
